source/manifest.json

```diff
@@ -16,6 +16,7 @@
 
   "permissions": [
     "activeTab",
+    "cookies",
```

Comment:
I have one question regarding this. We have only mentioned permission for `activeTab` while in the content script we are accessing all the tabs. Will it work fine ? As `activeTab` only give permission to access the active tab.  
Maybe adding `tabs` in permissions will be good ? 

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions#activetab_permission